pages/create-account.tsx

```diff
@@ -1,26 +1,39 @@
 import Head from 'next/head';
 import React, { useState } from 'react';
+import isEmail from 'validator/lib/isEmail';
+import isStrongPassword from 'validator/lib/isStrongPassword';
 import { useRouter } from 'next/router';
 import styles from '../styles/Home.module.css';
+import { Auth } from 'aws-amplify'
 
 export default function CreateAccount() {
   const [inputErrorMessages, setErrorMessages] = useState({
     confirmPassword: '',
+    email: '',
     firstName: '',
     lastName: '',
     password: '',
     username: '',
   });
 
+  const passwordConstraints = {
+    minLength: 8,        // Minimum length of 8 characters
+    minLowercase: 0,      // Minimum 0 lowercase letters
+    minUppercase: 0,      // Minimum 0 uppercase letters
+    minNumbers: 1,       // Minimum 1 numeric digits
+    minSymbols: 1,       // Minimum 1 special character
+  };
+
   const router = useRouter();
 
   const handleSubmit = async (e) => {
     e.preventDefault();
 
-    const userInputs = {};
-    const errors = {};
     const formData = new FormData(e.target);
 
+    const userInputs = {};
```

Comment:
Can remove as it looks like userInputs is no longer being used

Reply:
Should also be another line of userInputs underneath the `switch` statement as well.